diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/FileStructureRule.kt

```diff
@@ -73,7 +74,7 @@ class FileStructureRule(configRules: List<RulesConfig>) : DiktatRule(
         }
     private val refSet: MutableSet<String> = mutableSetOf()
     private var packageName = ""
-    private val ignoreImports = setOf("invoke", "get", "set", "getValue")
+    private val ignoreImportsPatterns = setOf("component\\d+".toRegex())
```

Comment:
May be add few comments, that this is not magic word, but relate to the multi declaration in kotlin, which uses `kotlin.collections / component*`?

Reply:
Added